psychopy/demos/coder/iohub/eyetracking/pupil_labs_debug.py

```diff
@@ -0,0 +1,48 @@
+from psychopy import logging
+from psychopy.core import getTime, wait
+from psychopy.iohub import launchHubServer
+
+logging.console.setLevel(logging.DEBUG)
+
+iohub_config = {
+    "eyetracker.hw.pupil_labs.pupil_core.EyeTracker": {
+        "name": "tracker",
+        "runtime_settings": {"pupil_capture_recording": {"enabled": False}},
+    }
+}
+
+io = launchHubServer(**iohub_config)
+
+# Get the eye tracker device.
+tracker = io.devices.tracker
+
+# run eyetracker calibration
+# r = tracker.runSetupProcedure()
+
+tracker.setConnectionState(True)
+tracker.enableEventReporting(True)
+
+# Check for and print any eye tracker events received...
+tracker.setRecordingState(True)
```

Comment:
`tracker.enableEventReporting(True)` and `tracker.setRecordingState(True)` do the same thing, so only one is needed. Since `tracker.setRecordingState(False)` is used later in the script, maybe remove the `tracker.enableEventReporting(True)`  call.

Reply:
In our implementation, they do not do the same but `setRecordingState` also calls `enableEventReporting`. This is why I agree with removing `enableEventReporting`.